pybliometrics/utils/create_config.py

```diff
@@ -33,23 +33,36 @@ def create_config(config_dir: Optional[Union[str, Path]] = None,
 
     # Set authentication
     config.add_section('Authentication')
+
+    # Get keys and tokens
+    new_keys, new_tokens = None, None
```

Comment:
Why are these two objects necessary? Why not simply use `keys` and 'tokens'?

Reply:
Let's simply use `keys` and `tokens`. I just wanted to avoid having one variable with several data types (`str` and `list[str]`) in the code.